modules/import_har.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+
+import apache_beam as beam
+from apache_beam.options.pipeline_options import PipelineOptions, StandardOptions
+
+from modules import constants, utils
+from modules.transformation import ImportHarJson, ReadHarFiles, WriteBigQuery
+
+
+def parse_args(argv):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--input',
+        dest='input',
+        # default='gs://httparchive/experimental/input/*',
+        help='Input file to process.')
+    parser.add_argument(
+        '--subscription',
+        dest='subscription',
+        # default='projects/httparchive/subscriptions/har-gcs-pipeline',
+        # default='projects/httparchive/subscriptions/har-gcs-experimental-sub',
```

Comment:
Yep, these were commented out during testing. I intend to keep the default for `--subscription` since we intend to run primarily in this mode. Will remove "experimental" comments.